graph/export.go

```diff
@@ -82,8 +82,12 @@ func (s *TagStore) CmdImageExport(job *engine.Job) error {
 	}
 	// write repositories, if there is something to write
 	if len(rootRepoMap) > 0 {
-		rootRepoJson, _ := json.Marshal(rootRepoMap)
-		if err := ioutil.WriteFile(path.Join(tempdir, "repositories"), rootRepoJson, os.FileMode(0644)); err != nil {
+		f, err := os.Create(path.Join(tempdir, "repositories"))
```

Comment:
nice catch, @ankushagarwal please us os.OpenFile in write mode then and specify the 0644 :)